spec/unit/inventory_v4_spec.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../spec_helper'
+
+include Hawkular::InventoryV4
+describe 'Inventory v4' do
+  let(:resource_hash) do
+    {
+      'id' => 'root',
+      'type' => {},
+      'children' => [
+        { 'id' => 'child-01', 'type' => { 'id' => 'type-01' }, 'children' => [] },
+        { 'id' => 'child-02', 'type' => { 'id' => 'type-02' },
+          'children' => [
+            { 'id' => 'child-03', 'type' => { 'id' => 'type-02' }, 'children' => [] },
+            { 'id' => 'child-04', 'type' => { 'id' => 'type-02' }, 'children' => [] },
+            { 'id' => 'child-05', 'type' => { 'id' => 'type-03' }, 'children' => [] }
+          ]
+        },
+        { 'id' => 'child-06', 'type' => { 'id' => 'type-01' }, 'children' => [] }
+      ]
+    }
+  end
+
+  let(:resource) do
+    Resource.new(resource_hash)
+  end
+
+  describe '#children' do
+    it 'should return direct children' do
```

Comment:
No action necessary on this right now, but we’re planning on standardize test names, per recommended on betterspecs.org. In this case, it’d be something like “returns direct children”.